.travis.yml

```diff
@@ -0,0 +1,33 @@
+osx_image: xcode9.4
+dist: xenial
+
+language: node_js
+node_js: 'node'
+
+os:
+    - osx
+    - linux
+    - windows
+    
+addons:
+    apt:
+        sources:
+            - sourceline: 'ppa:alexlarsson/flatpak'
+        packages:
+            - xvfb
+            - flatpak
+            - flatpak-builder
+            - elfutils
+            
+install:
+    - if [ $TRAVIS_OS_NAME = linux ]; then
+      flatpak --user remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo;
+      flatpak --user install -y org.freedesktop.Platform/x86_64/19.08 org.freedesktop.Sdk/x86_64/19.08 org.electronjs.Electron2.BaseApp/x86_64/stable;
```

Comment:
Linux now successfully builds -- the problem was using both `sudo` and `--user`, thus installing the Flatpaks to root.